frontend/src/components/Pages/MissionPage/MapPosition/MissionMapView.tsx

```diff
@@ -7,25 +7,31 @@ import NoMap from 'mediaAssets/NoMap.png'
 import { PlaceRobotInMap, PlaceTagsInMap } from '../../../../utils/MapMarkers'
 import { BackendAPICaller } from 'api/ApiCaller'
 import { TaskStatus } from 'models/Task'
+import { MapCompass } from 'utils/MapCompass'
 
 interface MissionProps {
     mission: Mission
 }
 
 const MapCard = styled(Card)`
     display: flex;
-    height: 600px;
-    width: 600px;
+    max-width: 600px;
     padding: 16px;
 `
 
 const StyledMap = styled.canvas`
     object-fit: contain;
-    max-height: 100%;
-    max-width: 100%;
+    max-height: 600px;
+    max-width: 540px;
```

Comment:
This will not scale to 90% all the time,no? if the container is smaller than 540 it will use all of it?